Mathlib/Data/Subtype.lean

```diff
@@ -179,6 +179,13 @@ theorem map_involutive {p : α → Prop} {f : α → α} (h : ∀ a, p a → p (
     (hf : Involutive f) : Involutive (map f h) :=
   fun x ↦ Subtype.ext (hf x)
 
+theorem map_ne {α β : Sort*} {p : α → Prop} {q : β → Prop} (f g : α → β)
```

Comment:
Perhaps you can use the `α β` that already exist instead of making local new ones?